src/components/TodoCount.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+    <div class="count-container">
+        <span class="todo-count">
+            총
+            <strong>{{ count }}</strong> 개
+        </span>
+        <ul class="filters">
+            <todo-filter v-for="filter in filters" :key="filter.className" :filterType="filter" />
```

Comment:
```suggestion
            <todo-filter 
                    v-for="filter in filters" 
                    :key="filter.className" 
                    :filterType="filter" 
            />
```
vue 공식문서나 강의를 보니 이런식으로 표현이 되있는데 한번 고려해보시면 좋을거같아요!